sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -1675,6 +1675,7 @@ pi_result cuda_piDeviceGetInfo(pi_device device, pi_device_info param_name,
   case PI_DEVICE_INFO_GPU_SLICES:
   case PI_DEVICE_INFO_GPU_SUBSLICES_PER_SLICE:
   case PI_DEVICE_INFO_GPU_EU_COUNT_PER_SUBSLICE:
+  case PI_DEVICE_INFO_GPU_HW_THREADS_PER_EU:
```

Comment:
For consistency, would you mind adding this case to pi_hip.cpp and pi_opencl.cpp as well?

Reply:
Sure, thanks!